### PR TITLE
clightd: 5.1 -> 5.2

### PR DESCRIPTION
**Pull request by eadwu**
###### Motivation for this change
Dependency of https://github.com/NixOS/nixpkgs/pull/115819#pullrequestreview-609249159

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 116136` at 16431814 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li><a href="https://gist.github.com/7431d7f0d103cda70ed00372dd753429">clight</a></li></li>
    <li><a href="https://gist.github.com/fa96de28294916610b3dcb181f5cbbca">clightd</a></li></li>
  </ul>
</details>
<details>
  <summary>3 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/build-tools-in-build-inputs.md">build-tools-in-build-inputs</a>

util-linux is a build tool so it likely goes to `nativeBuildInputs`, not `buildInputs`.

Near pkgs/applications/misc/clight/clightd.nix:48:3:
```
   |
48 |   buildInputs = with lib; [
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl3` is a deprecated license, check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/applications/misc/clight/clightd.nix:74:5:
```
   |
74 |     license = licenses.gpl3;
   |     ^
```
</li>
    <li>warning: unused-argument

Unused argument: `fetchpatch`.
Near pkgs/applications/misc/clight/clightd.nix:1:33:
```
  |
1 | { lib, stdenv, fetchFromGitHub, fetchpatch
  |                                 ^
```
</li>
  </ul>
</details>

File: pkgs/applications/misc/clight/clightd.nix [hunk 1]
```diff
@@ -9,39 +9,32 @@
...
   '';
 
   cmakeFlags = with lib;
-     optional enableDdc "-DENABLE_DDC=1"
+    [ "-DSYSTEMD_SERVICE_DIR=${placeholder "out"}/lib/systemd/system"
+      "-DDBUS_CONFIG_DIR=${placeholder "out"}/etc/dbus-1/system.d"
+      # systemd.pc has prefix=${systemd.out}
+      "-DMODULE_LOAD_DIR=${placeholder "out"}/lib/modules-load.d"
+    ]
+  ++ optional enableDdc "-DENABLE_DDC=1"
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/clight/clightd.nix:37]:
```suggestion
    ] ++ optional enableDdc "-DENABLE_DDC=1"
```

File: pkgs/applications/misc/clight/clightd.nix [hunk 2]
```diff
@@ -9,39 +9,32 @@
...
     owner = "FedeDP";
     repo = "Clightd";
     rev = version;
-    sha256 = "sha256-BEGum0t+FrCTAEQwTsWeYpoIqimTAz+Bv/ZQPQ3fePY=";
+    sha256 = "sha256-bb+PQsWLf4KnbzWUY6B42yHaDQKN9dtyfwSLe8FBaWU=";
   };
 
-  patches = [
-    # Not needed by next version bump
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/clight/clightd.nix:23]: Please remove fetchpatch from inputs.

Comment (eadwu) [reply pkgs/applications/misc/clight/clightd.nix:23]: Resolved.

**Pull request merged**